WebRoot/skins/_base/base4/skin.properties

```diff
@@ -181,7 +181,7 @@ InlineToolbarContainer      = border: 1px solid @SeparatorColor@; border-left:no
 
 TreeContainer               = @PanelContainer@
 TreeHeaderTextColor         = @hexToRGBA(TxtC, 80)@
-TreeHeaderContainer         = @TreeHeaderBg@  height: 3rem; @BottomSeparator@ padding-left: @SkinWrapperPadding@; padding-right: 8px; 
+TreeHeaderContainer         = @TreeHeaderBg@  height: 2.9166rem; @BottomSeparator@ padding-left: @SkinWrapperPadding@; padding-right: 8px; 
```

Comment:
Instead of reducing height to `35px`, can we have `@FixBoxModel@` ?

Reply:
Also you may also has to change `line-height` of property `@SearchToolbarHeader@` if we are going with this change.